CMakeLists.txt

```diff
@@ -106,13 +107,21 @@ ENDIF()
 
 include (CMakePackageConfigHelpers)
 
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/ZXingConfigVersion.cmake"
+    VERSION ${ZXing_VERSION}
```

Comment:
Could you please use `${PROJECT_VERSION}` instead. No need to introduce a new variable then.

Reply:
Done as requested. (And learned about `${PROJECT_VERSION}` …)

The generated `ZXingConfigVersion.cmake` file is exactly the same after this change and the tests results are as in the original PR.